### PR TITLE
Add ISTEXUnzip action, to unzip files from dl.istex.fr

### PR DESCRIPTION
**Pull request by parmentf**
Using dl.istex, one downloads a zip file containing JSON data.
This action is used to extract JSON data in a stream from the zip file.
See [Trello Card](https://trello.com/c/ndJ98lU9/227-ajouter-un-loader-de-fichier-zip-obtenu-avec-dlistexfr).

### CODE REVIEW

**Comment by parmentf** [comment]
Inspiration: https://github.com/touv/node-ezs/blob/master/src/statements/delegate.js

File: src/unzip.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                            }
+                            str += data2.toString();
+                            return feed2.end();
+                        }));
+                } else {
+                    entry.autodrain();
+                }
+            })
+            .on('error', e => feed.write(e))
+            .on('data', d => feed.write(d));
```
Comment (touv) [inline src/unzip.js]: to drop because unzipper emit 'entry' not 'data'

File: src/unzip.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+        const output = this.input
+            .pipe(unzipper.Parse())
+            .on('entry', (entry) => {
+                const fileName = entry.path;
+                if (fileName === 'manifest.json') {
+                    entry.autodrain();
+                } else if (fileName.endsWith('.json')) {
+                    let str = '';
+                    entry
```
Comment (touv) [inline src/unzip.js]: let str = '';
entry.on('data', (d) => {
   str += d.toString();
})
.on('end', (d) => {
 const obj = JSON.parse(str);
 feed.write(obj);
})

File: src/unzip.js (hunk 1 above)
Comment (parmentf) [reply src/unzip.js]: Weird, removing line 51 makes all tests timeout.

**Pull request merged**